bcda/lambda/optout/main.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/sirupsen/logrus"
+
+	bcdaaws "github.com/CMSgov/bcda-app/bcda/aws"
+	"github.com/CMSgov/bcda-app/bcda/database"
+	"github.com/CMSgov/bcda-app/bcda/models/postgres"
+	"github.com/CMSgov/bcda-app/bcda/suppression"
+	"github.com/CMSgov/bcda-app/bcda/utils"
+	"github.com/CMSgov/bcda-app/optout"
+
+	"github.com/CMSgov/bcda-app/conf"
+)
+
+func main() {
+	if os.Getenv("LOCAL_STACK_ENDPOINT") != "" {
```

Comment:
Reference number 1

Reply:
Added it here since it's the top of the file, so most likely to be seen by developers when perusing code.